main.go

```diff
@@ -96,6 +96,8 @@ func getLog(
 	// intentions...
 	writer.Header().Set("Content-Type", "text/plain; charset=utf-8")
 	writer.Header().Set("Access-Control-Allow-Origin", "*")
+	writer.Header().Set("Transfer-Encoding", "chunked")
```

Comment:
@eliperelman Thanks so much for writing this PR, I only just saw it - sorry for delay replying!

I believe, so long as we don't specify a `Content-Length` header, the go standard library should add/set the `Transfer-Encoding` header to `chunked` already... See [these docs](https://github.com/golang/go/blob/c007ce824d9a4fccb148f9204e04c23ed2984b71/src/net/http/httputil/httputil.go#L28-L30).

We should be able to test this though, to be sure.